src/coffee/directives/api/models/child/marker-child-model.coffee

```diff
@@ -23,7 +23,7 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         @trackModel = true, @needRedraw = false) ->
         #where @model is a reference to model in the controller scope
         #clonedModel is a copy for comparison
-        @clonedModel = _.extend({},@model)
+        @clonedModel = _.clone(@model,true)
```

Comment:
Regarding the clone/cloneDeep choice, cloneDeep might be more readable but I settled for clone as it seems it was the one used before that lodash dep' thing, at least this is what the following comment suggests (src/coffee/directives/api/models/child/marker-child-model.coffee l.83): 
```
#changed from _.clone(model, true) eliminates lodash dep (so you can use underscore)
``` 

Btw, I have no opinion on the lodash debate, w/e works (cloneDeep untested, though).
You might want to ask the contributor that made the switch to _.extend.